kyo-prelude/shared/src/main/scala/kyo/Var.scala

```diff
@@ -151,6 +151,62 @@ object Var:
     def runTuple[V, A: Flat, S](state: V)(v: A < (Var[V] & S))(using Tag[Var[V]], Frame): (V, A) < S =
         runWith(state)(v)((state, result) => (state, result))
 
+    object isolate:
+        abstract private[kyo] class Base[V: Tag] extends Isolate[Var[V]]:
+            type State = V
+            def use[A, S2](f: V => A < S2)(using Frame) = Var.use(f)
+            def resume[A: Flat, S2](state: State, v: A < (Var[V] & S2))(using Frame) =
+                Var.runTuple(state)(v)
+        end Base
+
+        /** Creates an isolate that sets the Var to its final isolated value.
+          *
+          * When the isolation ends, unconditionally updates the Var with the last value it had in the isolated computation.
+          *
+          * @tparam V
+          *   The type of value in the Var
+          * @return
+          *   An isolate that updates the Var with its isolated value
+          */
+        def update[V: Tag]: Isolate[Var[V]] =
+            new Base[V]:
+                def restore[A: Flat, S2](state: V, v: A < S2)(using Frame) =
+                    Var.set(state).andThen(v)
```

Comment:
If the tag was provided in the function itself, could we use a cached instance?

Reply:
It could be we'd need to extend `Isolate` to pipe the type parameter of the state, which would increase the complexity of the abstraction. I think it's ok to have the allocation since users can cache instances and reuse.